src/components/RecentOperationsCards/RecentOperationsCards.tsx

```diff
@@ -31,6 +25,7 @@ export interface IRecentOperationGroupCard {
 }
 
 export const RecentOperationDashboardCard: FC<IRecentOperationDashboardCard> = ({ item }) => {
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)